new/tk-mixlib-install-unification.md

```diff
@@ -0,0 +1,88 @@
+---
+RFC: unassigned
+Title: Test Kitchen Mixlib Install Unification
+Author: Patrick Wright <patrick@chef.io>
+Status: Draft
+Type: Standards Track
+---
+
+# Test Kitchen Mixlib Install Unification
+
+Test Kitchen currently supports two code paths for configuring, downloading and installing chef and chefdk on test instances. Both code paths use Mixlib Install but are implemented differently. In the past year, significant effort has been put into Mixlib Install to make it the canonical source for which all Chef related products are queried for meta-data and for generating the bootstrap installation scripts (install.sh and install.ps1). 
```

Comment:
...for interacting with Chef Software Inc's release infrastructure and generating bootstrap installation scripts (install.sh and install.ps1).